cmd/vela-s3-cache/restore.go

```diff
@@ -130,7 +149,7 @@ func (r *Restore) Validate() error {
 
 	// verify timeout is provided
 	if strings.EqualFold(r.Timeout.String(), "0s") {
```

Comment:
Same thing here.. I think we could simplify this to be:

```suggestion
	if r.Timeout == 0 {
```